recipe-server/client/control/sass/control.scss

```diff
@@ -151,15 +155,32 @@
     color: $red;
     display: block;
     padding-bottom: 5px;
+    text-transform: uppercase;
   }
 
-  /* Checkbox inputs should appear to the left of the label. */
-  .checkbox-field {
+  /* A group of related fields, generally for things like radio buttons. */
+  .fieldset {
+    @extend .form-field;
+    border: none;
+
+    .fieldset-label {
+      @extend .label;
+    }
+
+    /* Checkbox/radio fields within fieldsets should be inline in a row. */
+    .checkbox-field, .radio-field {
+      display: inline-flex;
+      margin-right: 1.5em;
+    }
+  }
+
+  /* Checkbox/radio inputs should appear to the left of the label. */
+  .checkbox-field, .radio-field {
```

Comment:
# CORNERS ARE FOR CUTTING

<sub><sup>No, I'll go and fix my linting errors. Sorry.</sup></sub>